src/utils/galileo-logger.ts

```diff
@@ -11,18 +9,28 @@ import {
   Trace,
   WorkflowSpan
 } from '../types/log.types';
+import {
+  LlmStepAllowedIOType,
+  RetrieverStepAllowedOutputType
+} from '../types/step.types';
 import { ProjectTypes } from '../types/project.types';
 
+class GalileoLoggerConfig {
+  public projectName?: string | undefined;
```

Comment:
The `?` already means it can be undefined so the ` | undefined` is redundant
```suggestion
  public projectName?: string;
```